torch/nn/parameter.py

```diff
@@ -1,7 +1,7 @@
-from torch.autograd import Variable
+import torch
 
 
-class Parameter(Variable):
+class Parameter(torch.Tensor):
```

Comment:
It's nice that we can inherit from `torch.Tensor` now (I had a segfault when I tried it last time, with a build from last week)